css-classes-completion/README.md

```diff
@@ -41,42 +41,42 @@ Atomic classes setting a text color or a background-color will get a color previ
 ### REM To PX Convertion
 
 This extension will automatically convert `rem` to `px` values as extra documentation comments.
-The convertion will be done using the `root font-size in pixel` setting value (default: 16)
+The conversion will be done using the `root font-size in pixel` setting value (default: 16)
 
 ![rem to px](media/doc/rem-to-px-feature.png)
 
 ### Hover Documentation
 
-Mouse over an utility class will display the related css property values applied.
+Mousing over a utility class will display the related CSS property values applied.
```

Comment:
I'm not sure about 'Mouse over' as a verb. What do you think about 'hovering your mouse over a utility class' ?